README.md

```diff
@@ -33,4 +33,4 @@ Workshop attendees and mentors can register for events at [FrontEndFoxes.org](ht
 
 The goal of the program is to familiarize women and those who identify as such with concepts of programming for web and mobile in a supportive and inclusive environment. Experienced developers, however, are very welcome to join in our efforts to organize events, support our scholarship fund, mentor a skulk, and provide content. We are grateful for all kinds of help!
 
-**Want to contribute? Please make sure to check out our [Code of Conduct](workshops/CODE_OF_CONDUCT.md).**
\ No newline at end of file
+**Want to contribute? Please make sure to check out our [Code of Conduct](workshops/CODE_OF_CONDUCT.md) as well as our [Contrubuting instructions](contributing.md).**
```

Comment:
one little fix: spelling of Contributing

Reply:
Thank you! I did the spelling fix. 👍